src/validation/permissions.js

```diff
@@ -0,0 +1,76 @@
+import Joi from '@hapi/joi';
```

Comment:
Please move the validation folder into the `helpers` folder. It makes the reference easier and file structure better.

Reply:
Done